### PR TITLE
Update TestsManifestGeneration.Etw.cs

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
Disable tests on x86

See https://github.com/dotnet/runtime/issues/97255

### CODE REVIEW

**Review by tommcdon — APPROVED** [review]
Thanks for all of the investigation on this issue Aaron!  I agree disabling the test is the right thing to do.

File: src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@ public partial class TestsManifestGeneration
         /// ETW only works with elevated process
         [ConditionalFact(nameof(IsProcessElevatedAndNotWindowsNanoServerAndRemoteExecutorSupported))]
         [SkipOnCoreClr("Test should only be run in non-stress modes", ~RuntimeTestModes.RegularRun)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(TestLibrary.PlatformDetection), nameof(TestLibrary.PlatformDetection.Is32BitProcess))]
```
Comment (AaronRobinsonMSFT) [inline src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs:34]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.Is32BitProcess))]
```

File: src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs [hunk 2]
```diff
@@ -71,6 +72,7 @@ public void Test_EventSource_EtwManifestGeneration()
 
         [ConditionalFact(nameof(IsProcessElevatedAndNotWindowsNanoServerAndRemoteExecutorSupported))]
         [SkipOnCoreClr("Test should only be run in non-stress modes", ~RuntimeTestModes.RegularRun)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(TestLibrary.PlatformDetection), nameof(TestLibrary.PlatformDetection.Is32BitProcess))]
```
Comment (AaronRobinsonMSFT) [inline src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs:75]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.Is32BitProcess))]
```

File: src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs [hunk 3]
```diff
@@ -31,6 +31,7 @@ public partial class TestsManifestGeneration
         /// ETW only works with elevated process
         [ConditionalFact(nameof(IsProcessElevatedAndNotWindowsNanoServerAndRemoteExecutorSupported))]
         [SkipOnCoreClr("Test should only be run in non-stress modes", ~RuntimeTestModes.RegularRun)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.Is32BitProcess))]
```
Comment (AaronRobinsonMSFT) [inline src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs:34]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.IsX86Process))]
```

File: src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs [hunk 4]
```diff
@@ -71,6 +72,7 @@ public void Test_EventSource_EtwManifestGeneration()
 
         [ConditionalFact(nameof(IsProcessElevatedAndNotWindowsNanoServerAndRemoteExecutorSupported))]
         [SkipOnCoreClr("Test should only be run in non-stress modes", ~RuntimeTestModes.RegularRun)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.Is32BitProcess))]
```
Comment (AaronRobinsonMSFT) [inline src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.Etw.cs:75]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/97255", typeof(PlatformDetection), nameof(PlatformDetection.IsX86Process))]
```

**Pull request merged**